lib/tasks/update_resource_quantity.rake

```diff
@@ -0,0 +1,14 @@
+namespace :data do
+  desc 'Update resources with nil quantity'
+  task update_resource_quantity: :environment do
+    Resource.all.each do |resource|
+      resource.used = 0 unless resource.used.present?
+      resource.quantity = resource.used unless resource.quantity.present?
+      if resource.save
+        puts 'Success for resource with id ' + resource.id.to_s + ' and name ' + resource.name
+      else
+        puts 'Failed for resource with id ' + resource.id.to_s + ' and name ' + resource.name
+      end
+    end
+  end
+end
```

Comment:
Whenever you see this in the code you push in github, please fix it. You can configure your text editor to do this automatically, so that you never have to care for this again. Ping me on IRC to help you enable it.